src/api/events/webcontents.ts

```diff
@@ -21,4 +21,7 @@ export interface WindowResourceResponseReceivedEvent<Topic, Type> extends Window
 export interface WebContentsEventMapping<Topic = string, Type = string> extends BaseEventMap {
     'resource-load-failed': WindowResourceLoadFailedEvent<Topic, Type>;
     'resource-response-received': WindowResourceResponseReceivedEvent<Topic, Type>;
+    'did-change-theme-color': WindowEvent<Topic, Type>;
```

Comment:
Do you need to add these to Window's and View's propagated maps?

Reply:
I'm handling propagation in a separate PR once all the PRs are in.

Reply:
I'm handling Propagation in a separate PR once all PRs are in.